### PR TITLE
warning email

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

**Review by fangfy — COMMENTED** [review]
If it doesn't raise an exception, the code will continue to the next section and would that fail (`for obj in response["Contents"]`)?

File: dags/utils/landsat_scenes_identifying_logic.py [hunk 1]
```diff
@@ -148,9 +149,14 @@ def build_asset_list(scene):
...
+                task_name=f'Landsat Identifying - {scene["Date Product Generated L2"]}',
+                warning_message=f"Error Listing objects in S3 {USGS_S3_BUCKET_NAME} -"
+                f" folder {folder_link} - response {response}",
             )
+            # raise Exception(
+            #     f"Error Listing objects in S3 {USGS_S3_BUCKET_NAME} folder {folder_link}"
+            # )
 
         mtl_sr_st_files = {}
         for obj in response["Contents"]:
```
Comment (fangfy) [inline dags/utils/landsat_scenes_identifying_logic.py:162]: This is the section I'm wandering. Should you also have a line to skip process the response further?

Comment (carioca-au) [reply dags/utils/landsat_scenes_identifying_logic.py]: you are totally right ... fixed

**Comment by carioca-au** [comment]
> the code will continue to the next section and would that fail (`for obj in response["Contents"]`)?

Totally right ..... Fixed

**Review by fangfy — APPROVED** [review]
Looks OK to me if tested to work.

**Pull request merged**